app/components/highlight/module.js

```diff
@@ -0,0 +1,211 @@
+var highlightModule = (function() {
```

Comment:
Ah, so this is part of the module.js file. I created it during our last meeting when I was scrambling to change the "module" into a real module. This file should have been deleted before making the request. Completely my bad.